etc/conf/options.json

```diff
@@ -1,9 +1,14 @@
 {
     "broker_uri": "amqp://10.10.2.1:30000",
+    "zipkin_host": "",
```

Comment:
Put a valid zipkin host and port. Otherwise, I'd suggest to put a field named `zipkin_uri`  and validate the uri as well as match the host and port fields with a regular expression as you can find [here](https://github.com/labviros/is-aruco-detector/blob/5293af6be33cd7b396981c8ec5e9ce2d84c0ef05/src/is/aruco-service/service.cpp#L54). Despite being a C++ implementation, you can use the same regular expression with `re` python library.

Reply:
I going to leave the implementaton of the regular expression to the next PR.